docs/configuration/monitoring.md

```diff
@@ -1,77 +1,72 @@
 
-Currently two monitoring solutions are supported:
+Currently, we support two monitoring solutions:
 
-* [**Crow**](#setting-up-monitoring-using-crow) - A simple, lightweight, and built-in monitoring solution.
-* [**Prometheus + Grafana + Loki**](#setting-up-monitoring-using-grafana-prometheus-and-loki) - A comprehensive third-party monitoring solution, recommended for more advanced usage.
+-   **[Crow](#setting-up-monitoring-using-crow):** A simple, lightweight, and built-in monitoring solution.
+-   **[Prometheus + Grafana + Loki](#setting-up-monitoring-using-grafana-prometheus-and-loki):** A comprehensive third-party monitoring solution, recommended for more advanced usage.
+
+The sections below provide details on how to set up each monitoring solution.
 
 ## Setting up monitoring using Crow
 
-Crow displays information about the projects pending analysis and the jobs currently running on Codacy.
+Crow displays information about the projects that are pending analysis and the jobs currently running on Codacy.
 
-The Crow tool is installed alongside Codacy after the helm chart is deployed to the cluster.
-It can be accessed through the `/monitoring` path of the url pointing to your Codacy
-installation, e.g. `http://codacy.company.org/monitoring`. You must set the
-`global.codacy.crow.url` value in your `values.yaml` file so that anchor links to your
-projects can be properly established inside `crow`.
+The Crow tool is installed alongside Codacy when the Helm chart is deployed to the cluster and is configured as follows by default:
 
-For example:
+-   Crow is available on the `/monitoring` path of your Codacy installation URL, such as `http://codacy.company.org/monitoring`
+-   The default credentials to access Crow are:
 
-```yaml
-global:
-  codacy:
-    crow:
-      url: "http://codacy.example.com/monitoring"
-```
+    ```yaml
+    username: codacy
+    password: C0dacy123
+    ```
 
-### Configuring your credentials
+Follow the steps below to configure Crow and change the default configurations:
 
-We highly recommend that you define a custom password for the Crow installation. You can do this either through the `values.yaml` file or through a `--set` parameter during the Helm installation process. This parameter can be configured as follows:
+1.  Set the `global.codacy.crow.url` value in your `values.yaml` file so that Crow correctly generates anchor links to your projects. For example:
 
-* Through a `--set` parameter:
+    ```yaml
+    global:
+      codacy:
+        crow:
+          url: "http://codacy.example.com/monitoring"
+    ```
 
-```yaml
-helm upgrade (...options used to install codacy...) --set crow.config.passwordAuth.password=<--- crow password --->
-```
+2.  Set the `crow.config.passwordAuth.password` value in your `values.yaml` file to define a custom password for Crow:
 
-* Through the `values.yaml` file:
+    ```yaml
+    crow:
+      config:
+        passwordAuth:
+          password: <--- Crow password --->
+    ```
```

Comment:
On this part of the `.yaml` file we have a configuration section for Crow that includes a value for setting the PostgreSQL user name and password. Shouldn't the user configure these values as well at this time?

```yaml
crow:
  replicaCount: 1
  fullnameOverride: codacy-crow
  crowdb:
    create: false
    host: codacy-crowdb
    postgresqlUsername: "codacy"
    postgresqlDatabase: "crow"
    postgresqlPassword: "PLEASE_CHANGE_ME"
    service:
      port: 5432
    fullnameOverride: codacy-crowdb
  config:
    play:
      http:
        context: "/monitoring"
    passwordAuth:
      password: "PLEASE_CHANGE_ME"
```

Reply:
Yes, that's a good idea.

Reply:
Added explicit instructions for this (https://github.com/codacy/chart/pull/339/commits/90c801b1f8afbcf48a628e9181cbb22640cd8e2c).